config/manifests/bases/clusterlogging.clusterserviceversion.yaml

```diff
@@ -9,6 +9,16 @@ metadata:
     containerImage: quay.io/openshift-logging/cluster-logging-operator:latest
     description: The Red Hat OpenShift Logging Operator for OCP provides a means for
       configuring and managing your aggregated logging stack.
+    features.operators.openshift.io/cnf: "false"
+    features.operators.openshift.io/cni: "false"
+    features.operators.openshift.io/csi: "false"
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "true"
```

Comment:
I don't think we can claim fips-compliant=true without fips scan. 

According the latest scan result. we have to fix the issue below
```
logging-view-plugin-container | /opt/app-root/plugin-backend | go binary is not CGO_ENABLED 
log-file-metric-exporter-container | /usr/local/bin/log-file-metric-exporter | go binary has no build tags set (should have strictfipsruntime) 
cluster-logging-operator-container | /usr/bin/cluster-logging-operator | go binary has no build tags set (should have strictfipsruntime) | 
```
The company only provide a tool to scan golang and java based image.  No idea how to scan fluentd? No idea how to scan vector?

Reply:
@anpingli  I would like to emphasize our criteria for true/false is based upon the provided definition of the annotation:
```
Whether the opperator accepts the [FIPS-140](https://issues.redhat.com/browse/FIPS-140) configuration of the underlying platform and works on nodes that are booted into FIPS mode. In this mode, the operator and any workloads it manages (operands) are solely calling the RHEL cryptographic library submitted for [FIPS-140](https://issues.redhat.com/browse/FIPS-140) validation
```
If scanning is the only way to confirm :arrow_up: then we need the scan, but I do not believe this is what it says

Reply:
My understanding is this text accurately represents "our" definition, and needs to be set to true in order for our operator to show in the filtered catalog list.

Reply:
I also think we need to set this to true and work on the scan in the background if we need to.

Reply:
I think we should use fips-compliant=false before the scan issue to be fixed.  https://issues.redhat.com/browse/LOG-4886

Reply:
The QE/ART/Security team is planning to scan the operator according to fips-compliant.  it may block the release if scan failed when fips-compliant=true